### ISSUE TITLE
Broken rendering of verbatim section

### ISSUE THREAD

**Issue by andk** [CPAN-API/metacpan-web#1664]
Today I see a broken rendering of the SYNOPSIS section on the page https://metacpan.org/pod/HTML::MyHTML. The current version of the distro is 0.33.

Broken is that the first character on every line is missing, so instead of 'use utf8' I only see displayed 'se uft8', the leading 'u' is missing.

The brokenness seems only to affect this single verbatim section, all others look OK.

**Comment by haarg** [comment]
Does this happen every time you load the page?  Can you scroll the verbatim section horizontally?

**Comment by andk** [comment]
Well spotted! This is embarrassing, I did not try to scroll horizontally because I did not see the bottom edge of the code division. But I could have scrolled, I'm sure. I suppose I did cause the scrolling to the right by one character somehow without noticing. At least I cannot reproduce the broken page without careful horizonal scrolling.

I think I better close this ticket now and try to remember how to deal with such a rare surprise.

Thanks!!!
